lms/djangoapps/ora_staff_grader/tests/test_serializers.py

```diff
@@ -529,3 +530,44 @@ def test_happy_path(self):
 
         data = LockStatusSerializer(self.lock_owned_by_other_user).data
         assert data == self.lock_owned_by_other_user_expected
+
+
+class TestStaffAssessSerializer(TestCase):
+    """ Tests for StaffAssessSerializer """
+    grade_data = {
+        # TODO - determine if this is empty or if key is nonexistant for no feedback
```

Comment:
@muselesscreator - what are behaviors of `overallFeedback` for both (1) feedback disabled and (2) feedback enabled but not provided?

Reply:
No strong feelings.  Probably null if disabled and empty string if not provided?